learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -248,63 +267,67 @@ Accessing Scene Nodes
 ~~~~~~~~~~~~~~~~~~~~~
 
 One of the most common tasks is accessing Scene Tree Nodes (again, not to mistake with *Visual Script Nodes*).
-Dragging from the Scene Tree and dropping into the canvas, by default, will ask you to *call a method* (sometimes referred to as *member function*) on
-this node. 
+Dragging from the Scene Tree and dropping into the canvas, by default, will ask you to *call a method* (sometimes referred to as *member function*) on this node. 
```

Comment:
Too many commas. If it happens, it's the default and doesn't need to be called out. 
 "dropping into the canvas will"